jenkins/Jenkinsfile

```diff
@@ -162,6 +163,8 @@ pipeline {
           images.add('leonardo-jupyter')
           images.add('welder-server')
           images.add('openidc-proxy')
+          images.add('anvil-rstudio-base')
+          images.add('anvil-rstudio-bioconductor')
```

Comment:
I'm thinking maybe we should make a map of tool -> customImage, and have separate image for RStudio vs jupyter to save some space

Reply:
Yeah that might make sense going forward, there are a lot of images here..

Reply:
I feel like I'd want to do that in a separate PR though. I could also leave out the anvil-rstudio images for now..

Reply:
We could also theoretically generalize the terra-docker job/conf to have a repo/dir specification for each image, or include the images as a submodule to allow them to be built and scanned like the other images if we want to have more ownership

This would allow that job to update the reference.conf above and cascade updates to the custom dataproc image. Might be worth a follow-up ticket to cut down on future manual updating. As of now, an update to the reference.conf will cause the new images to be built into the custom image though, so it may not be super necessary